### DISCUSSION TITLE
[QOL] Rune pages name syntax

### DISCUSSION THREAD

**Discussion by Tr3c3** [cangzhang/ChampR#236] [Ideas]
It would be helpful if the champion name was included at the beginning of a rune page name, i.e. "[Lolalytics] Most common@Bottom, 12.10 (Diamond+)" could be "[Lolalytics] Tristana Most Common@Bottom". This way, most accidents related to the reliance on the app and its manual operation may be solved.

On the same line (not sure if this needs to be put as a separate suggestion, will do if instructed to), it'd be nice to have a setting to automatically import a champion's runes whenever a champion is either locked in or traded with another player or AR modes' bench. 
 
The role picked would be retrieved from the matchmaking assignation (in draft queues) or the most played one for blind queues, and we could pick a default web source and type (most common or highest winrate).

Sample picture:
![image_2022-06-08_214350488](https://user-images.githubusercontent.com/8467335/172740650-f49c1ed4-4543-4d87-8274-c4b17fd2306f.png)

**Comment by cangzhang** [comment]
Hi Sebastián, your suggestion makes sense. I will make change to rune name in several days.

As for the second, not every data source has its most common/highest win builds/runes, so I will have an option in settings to automatically set first rune (usually it's the most common one) from certain source.

**Comment by httpsColon** [comment]
您吉祥，lord sheep
